source/acquaman/IDEAS/IDEASScanConfiguration.h

```diff
@@ -0,0 +1,92 @@
+#ifndef IDEASSCANCONFIGURATION_H
+#define IDEASSCANCONFIGURATION_H
+
+#include "acquaman/IDEAS/IDEASScanConfigurationDbObject.h"
+#include "application/IDEAS/IDEAS.h"
+
+/*!
+	This is a non-QObject class that holds the database object for a general
+	scan configuration.  It has getters and setters to the database object.
+	*/
+class IDEASScanConfiguration
```

Comment:
Just curious what the motivation was for this separation of ScanConfiguration from its DbObject version is?